fe/fe-core/src/main/java/com/starrocks/clone/TabletScheduler.java

```diff
@@ -615,7 +615,8 @@ private void scheduleTablet(TabletSchedCtx tabletCtx, AgentBatchTask batchTask)
         }
     }
 
-    private void handleTabletByTypeAndStatus(TabletStatus status, TabletSchedCtx tabletCtx, AgentBatchTask batchTask)
+    // 'public' for test only
+    public void handleTabletByTypeAndStatus(TabletStatus status, TabletSchedCtx tabletCtx, AgentBatchTask batchTask)
```

Comment:
You can use @VisibleForTesting annotation from guava to mark this situation.

Reply:
done